packages/core/src/extensions/as-legacy-globals-for-extension-api/as-legacy-global-singleton-object-for-extension-api.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import type { Injectable } from "@ogre-tools/injectable";
+import { asLegacyGlobalForExtensionApi, getLegacyGlobalDiForExtensionApi } from "@k8slens/legacy-global-di";
+import { loggerInjectionToken } from "@k8slens/logger";
+
+export interface LegacySingleton<T> {
+  createInstance(): T;
+  getInstance(): T;
+  resetInstance(): void;
+}
+
+export function asLegacyGlobalSingletonForExtensionApi<Instance>(injectable: Injectable<Instance, unknown, void>): LegacySingleton<Instance>;
```

Comment:
Why is this needed? Should this be part of `@k8slens/legacy-global-di`?

Reply:
this was an effect of the merge